test/mpi/coll/customreduce.c

```diff
@@ -0,0 +1,102 @@
+#include "mpi.h"
```

Comment:
There is a copyright header that needs to be included with all files in the repo. Can you make sure to add it here?